README

```diff
@@ -40,22 +41,22 @@ Sharing personal word lists between spell-checkers
 It is possible, and usually safe, to share Enchant’s personal word lists
 with other spelling checkers that use the same format (a simple plain text
 file with one word per line). The spell-checkers known to be compatible are
-Hunspell and Ispell. (Although Enchant does not support Ispell as a
+Nuspell, Hunspell and Ispell. (Although Enchant does not support Ispell as a
```

Comment:
Please use the same order "Hunspell, Nuspell" here, and in the rest of this file (including example spellchecker wordlist filenames).

Reply:
Fixed in https://github.com/nuspell/enchant/commit/205c4b8423d3140a1c6f3e3e77f0841145c7fb69#diff-c47c7c7383225ab55ff591cb59c41e6b